tools/crm_attribute.c

```diff
@@ -535,6 +539,12 @@ main(int argc, char **argv)
     }
 
 done:
+    free(options.attr_id);
+    free(options.attr_name);
+    free(options.attr_pattern);
```

Comment:
Don't free attr_pattern, if it's non-NULL attr_name will point to the same memory

Reply:
Oh, sneaky.  Good catch.